_posts/2019-01-23-lecture-03.md

```diff
@@ -0,0 +1,471 @@
+    </div>
+</figure>
+
+* In (a) and (b), the shaded $$Y$$ node blocks the ball from going between nodes $$X$$ and $$Z$$. This gives the independence relation that was introduced in the last lecture: $$X \perp Z \mid Y$$.
+
+* (c), also called the "V-structure", is a special case. Opposite from the first two exmaples, the ball can go between $$X$$ and $$Z$$ if the node $$Y$$ is shaded, and is blocked otherwise. Therefore, the graph on the right yields $$X \perp Z$$.
+
+With these basic structures, we can apply the rules on a DAG. For example, let us try to find whether $$X_2$$ and $$X_3$$ are independent given $$X_1$$ and $$X_6$$.
+
+<figure id="bayes-ball-example" class="l-body">
+    <div class="row">
+        <div class="col two">
+            <img src="{{ 'assets/img/notes/lecture-03/dag.png' | relative_url }}" />
+        </div>
+    </div>
+</figure>
+
+After shading $$X_1$$ and $$X_6$$, the ball cannot go from $$X_2$$ to $$X_3$$ through $$X_1$$ because it is blocked; however, $$X_2$$, $$X_6$$, and $$X_5$$ forms a "V-structure", so the ball can go along the path $$X_2$$, $$X_6$$, $$X_5$$, $$X_3$$. Therefore, the independence statement is invalid.
+
+## Limits of Directed and Undirected GMs
+
+From a representational perspective, we aim to find a graph $$\mathcal{G}$$ that precisely captures the independencies in a given distribution $$P$$. This goal of learning GMs motivates the following definition.
+
+### Perfect Maps
+
+We say that a graph $$\mathcal{G}$$ is a **perfect map (P-map)** for a set of independencies $$\mathcal{I}$$ if $$\mathcal{I}(\mathcal{G}) = \mathcal{I}$$. We say that $$\mathcal{G}$$ is a perfect map for $$P$$ if $$\mathcal{I}(\mathcal{G}) = \mathcal{I}(P)$$. That is,
+$$
+\text{sep}_{\mathcal{G}}(X;Z \vert Y) \iff P \models (X \perp Z \vert Y)
+$$.
+
+* The P-map of a distribution is unique up to I-equivalence between networks. That is, a distribution P can have many P-maps, but all of them are I-equivalent.
+
+Arbitrary distribution $$P$$'s, however, do not necessarily attain perfect maps as either undirected or directed GMs. Two such examples are shown below. 
+<figure id="graph-separation" class="l-body">
+  <div class="row">
+    <div class="col two">
+      <img src="{{ 'assets/img/notes/lecture-03/graph_venn.png' | relative_url }}" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>A Venn diagram illustration of graphical model families, where D is the family of directed GMs, and U the family of undirected GMs.</strong>
+  </figcaption>
+</figure>
+
+<figure id="graph-separation" class="l-body">
+  <div class="row">
+    <div class="col two">
+      <img src="{{ 'assets/img/notes/lecture-03/graph_counterexamples.png' | relative_url }}" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>**Left**: A distribution with no possible DGM representation, which entails $$A\perp C\vert \{B,D\}$$ and $$B\perp D\vert \{A,C\}$$. **Right**: The v-structure is a distribution with no UGM representation.</strong>
```

Comment:
Sorry, it seems the **bold** and $$math$$ formatting don't work here. Maybe you can just put the text outside, like this?
`
```suggestion
 **Left**: A distribution with no possible DGM representation, which entails $$A\perp C\vert \{B,D\}$$ and $$B\perp D\vert \{A,C\}$$. **Right**: The v-structure is a distribution with no UGM representation.
```

Reply:
Can you make the suggested change? Otherwise the **bold** and $$math$$ formatting doesn't work:
![screen shot 2019-01-31 at 9 45 01 pm](https://user-images.githubusercontent.com/2617555/52099507-8d2cda00-25a1-11e9-8f5d-bda8c31f2c4f.png)